openmls/src/config/mod.rs

```diff
@@ -224,3 +224,20 @@ impl CiphersuiteName {
         false
     }
 }
+
+/// Check if all extension and proposal types are supported.
+pub(crate) fn check_required_capabilities_support(
```

Comment:
It would be good if you could add some comments on the [wiki page](https://github.com/openmls/openmls/wiki/Message-validation), even if it's just a quick note. I can later bring it into shape and integrate it with the rest of the validation scheme.

Reply:
https://github.com/openmls/openmls/wiki/Message-validation#required-capabilities